SPECS/swig/swig.spec

```diff
@@ -21,6 +30,7 @@ interpreted programming environments, systems integration, and as a
 tool for building user interfaces
 
 %prep
+pip3 install 2to3
```

Comment:
I think this might not be the right approach.
The regular build won't be happy about it IMO.

Reply:
Correct, there's no Internet access at that time. If anywhere, I'd put it inside the `%check` section. Btw. I've seen issues with missing `2to3` in `python-suds` or `python-tempita` and it looked like updating the package helped (the dropped their requirement in the later versions) - I think this tool is getting deprecated. I'm not saying the update is necessary here, it just might be another way to get around the issue.

Reply:
This tool is already the latest version. The problem in installing in check section, the tools is required during config itself. Based on the config result test files are built with py3 extension. To avoid build failure, let me put it inside with_check condition.  Or plz let me know whether there is a better to way to fix runtime dependencies like this.

Reply:
I think `%if %{with_check}` approach is great :)